src/utils.ts

```diff
@@ -366,6 +366,9 @@ export function getPermitTotalPrice(
 }
 
 export function isValidIBAN(value: string): boolean {
+  if (!value || value === '') {
```

Comment:
Is `value===''` redundant here?

However depending on whether the string contains spaces this could be

```javascript

if(!value || value.trim() === '') {
```